jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -5870,10 +5837,9 @@ public static Class<?> forPrimitiveName(String typeName) {
 		};
 	}
 
-	Method findMethod(boolean isPublic, String methodName, Class<?>... parameterTypes) {
+	Method findMethod(boolean publicOnly, String methodName, Class<?>... parameterTypes) {
 		try {
-			return getMethodHelper(true, false, null, isPublic ? SearchMethodType.PUBLIC : SearchMethodType.NOTPUBLIC,
-					methodName, parameterTypes);
+			return getMethodHelper(true, false, publicOnly, null, methodName, parameterTypes);
```

Comment:
The first parameter should be `false` to avoid the cost of throwing and catching the exception.

Reply:
Updated.